index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e520ead38a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="header">
+        <nav class="navbar">
+            <div class="web-name">Sohail</div>
+            <a href="#" class="bar-button">
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </nav>
+    </div>
```

Comment:
- [ ] Kindly consider using semantic tags for better code readability :mag:, user experience, and greater accessibility. In this case, you can use a `header` tag instead of a `div.` :smile_cat: